onepager.py

```diff
@@ -55,31 +55,29 @@
 
 
 # Strings (str)
-standard_string = "string"
-standard_string = 'string'
+std_string = "string"
 standard_char = "c"                             # No char type, only string of length 1
 standard_char = 'c'
 a_string: str = "string"                        # Typehint, optional
 a_string: str = 'string'
 
 concat_string = "string" + "c"                  # "stringc"
-ss_char = standard_string[0]                    # "s"
+ss_char = std_string[0]                         # "s"
 format_string1 = f"format {ss_char}"            # "format s" 
 format_string1 = f"format {pos_int}"            # "format 128" 
 format_string2 = "format %s" % ss_char          # "format s"
 format_string2 = "format %d" % pos_int          # "format 128"
 format_string3 = "format {}".format(ss_char)    # "format s"
 format_string3 = "format {}".format(pos_int)    # "format 128"
-standard_string.encode('utf-8')                 # b'string', encoded to bytes with utf-8 codec
-type(standard_string) == str                    # = True
 
+# Encoding (translation from characters (utf-8 or otherwise) to bytes) / Decoding (bytes to characters)
+type(std_string) == str                         # = True
+std_string = std_string.encode('utf-8')         # = b'string', encoded to bytes with utf-8 codec. Value is only returned
```

Comment:
**suggestion**: with this model you'd probably want to at least name `std_string` `std_bytes` because the encoding string is now a byte string. In Python 3 this is a really important distinction.

Reply:
Okay, see changes.